### PR TITLE
add get cloud-tenant endpoint

### PR DESCRIPTION
**Pull request by ariquinones**

### CODE REVIEW

File: src/sdk/model/json/cloud-tenant.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  contract_uuid: string;
+  location_id: string;
+  endpoint: string;
+  backup_count: number;
+}
+
+/**
+ * Interface for VeeamResource JSON properties.
+ */
+export interface VeeamResource {
```
Comment (cfsnyder) [inline src/sdk/model/json/cloud-tenant.ts]: all of these interfaces that describe JSON formats should have names that end in `Json`

File: src/sdk/model/cloud-tenant.ts [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+   */
+  get enabled(): boolean {
+    return this._json.enabled;
+  }
+
+  /**
+   * Gets the VeeamResource of the Cloud Tenant.
+   * @returns {VeeamResource} value
+   */
+  get resources(): VeeamResource {
```
Comment (cfsnyder) [inline src/sdk/model/cloud-tenant.ts]: There is no reason to make the javascript/typescript representation match the representation in the JSON. In fact, one of the main reasons for writing these wrapper classes is to make the Javascript/Typescript client interface better than the interface provided by the API. This is a perfect case actually.

Comment (ariquinones) [reply src/sdk/model/cloud-tenant.ts]: @cfsnyder pr updated

**Comment by cfsnyder** [comment]
Thanks @ariquinones !

**Pull request merged**